Source/Evolve/WarpXEvolve.cpp

```diff
@@ -251,15 +252,15 @@ WarpX::Evolve (int numsteps)
             bool const reset_fields = true;
             ComputeSpaceChargeField( reset_fields );
         }
-
-        amrex::Print()<< "STEP " << step+1 << " ends." << " TIME = " << cur_time
-                      << " DT = " << dt[0] << "\n";
-        Real walltime_end_step = amrex::second();
-        walltime = walltime_end_step - walltime_start;
-        amrex::Print()<< "Walltime = " << walltime
-                      << " s; This step = " << walltime_end_step-walltime_beg_step
-                      << " s; Avg. per step = " << walltime/(step+1) << " s\n";
-
+        if (verbose) {
+            amrex::Print()<< "STEP " << step+1 << " ends." << " TIME = " << cur_time
+                        << " DT = " << dt[0] << "\n";
+            Real walltime_end_step = amrex::second();
+            walltime = walltime_end_step - walltime_start;
```

Comment:
@MKieburtz I think it would be better to put `walltime_end_step` and the walltime modification above the `if` block, so we only omit print statements, in case someone uses walltime elsewhere.